frontend/lib/src/components/widgets/DataFrame/DataFrame.tsx

```diff
@@ -337,6 +337,7 @@ function DataFrame({
 
   return (
     <StyledResizableContainer
+      data-testid="stDataFrameContainer"
```

Comment:
nit: prefer `stDataFrame` to coincide with `className`

Reply:
done!